docs/sections/settings.rst

```diff
@@ -254,3 +254,17 @@ RECOMMENDED. ``list``. List containing a list of the Claim Names of the Claims t
 https://openid.net/specs/openid-connect-discovery-1_0.html#ProviderMetadata
 
 Default is an empty list as this application will not make assumptions about what the provider will support.
+
+OIDC_RSA_CERT_STORE
+===================
+
+RECOMMENDED: A ``str``, one of ``['database', 'filesystem']``
+
+If this is set to ``'database'`` RSA certs will be stored in the database.
+If this is set to ``'filesystem'`` they will be stored in ``MEDIA_ROOT/oidc_provider/rsa_keys/``.
```

Comment:
`MEDIA_ROOT` is for user uploaded files as in users of the website not the administrators, it can be configured to be publicly exposed with the `MEDIA_URL` setting. So it's probably not an appropriate place to be storing secret keys

Reply:
Fix: https://github.com/catalyst/django-oidc-provider/pull/14